R/bm-tpc-h.R

```diff
@@ -135,6 +135,18 @@ tpc_h <- Benchmark("tpc_h",
       ( params$engine == "dplyr" & params$format == "native" )
     params[!drop,]
   },
+  batch_id_fun = function(params) {
+    batch_id <- uuid()
+    paste0(batch_id, "-", params$scale_factor, substr(params$format, 1, 1))
```

Comment:
I know this is effectively copy|paste from voltrondata-labs/benchmarks but ouch that's kinda gross. Would you mind adding a todo here to make this be a bit nicer (probably need to properly stringify the scale factor and also have the format be full(er)?)

Reply:
Sure. We could also change it now, if you like; the only thing I can think of that might (? not sure) care about consistency here is @boshek's release report.

Reply:
Put in a `TODO` for now